src/isolate.rs

```diff
@@ -9,17 +9,19 @@ use errors::DenoError;
 use flags;
 use libdeno;
 
+#[allow(unused_imports)] // TODO: figure out how to remove this.
```

Comment:
@ry do you know how to do this?
If I remove `use futures` then rust errors on the use of `futures::future::ok` below.

Reply:
Oh sorry - forgot about that comment. Move it inside the `mod test` block.